db/migrate/20250103234408_create_organizations.rb

```diff
@@ -0,0 +1,10 @@
+class CreateOrganizations < ActiveRecord::Migration[7.2]
+  def change
+    create_table :organizations do |t|
+      t.string :name
```

Comment:
Since the validation for :name is :presence: true in Organization model
should the null: false be set in the Organizations table?
```suggestion
      t.string :name, null: false
```